### ISSUE TITLE
Pretrained model?

### ISSUE THREAD

**Issue by voa18105** [rosinality/style-based-gan-pytorch#3]
Can you / will you upload any pretrained model to compare results, please?

**Comment by rosinality** [comment]
https://drive.google.com/file/d/1zmVRIXk8HHddLmibKcEYpT0gixUfPlSp/view?usp=sharing

You can use this. 600k checkpoint of generator.

**Comment by voa18105** [comment]
Awesome! Thank you, man! I will tell you in a couple of days (hopefully) if I've managed to reproduce your results (reach more or less same quality)

**Comment by Johnson-yue** [comment]
> Awesome! Thank you, man! I will tell you in a couple of days (hopefully) if I've managed to reproduce your results (reach more or less same quality)

Did you use pretrained model？？

**Comment by voa18105** [comment]
@Johnson-yue Yes, it works good

**Comment by voa18105** [comment]
@rosinality  I've reached more or less same result. Did you try CelebA-HQ? How do you think if I will increase number of layer - will it generate HQ images?

**Comment by rosinality** [comment]
@voa18105 I didn't tried, but I think it will work - network architecture is almost same. But you will also need to modify train script, as I hard coded things a lot. :/

**Comment by voa18105** [comment]
@rosinality thanks, I think I am gonna try it nearest days

**Comment by anuragranj** [comment]
Hi,
Would it be possible to make the discriminator weights available too? I would like to be able to fine tune the model.
Thanks so much :)

**Comment by rosinality** [comment]
@anuragranj https://drive.google.com/file/d/1SFn7GygaLYhOobNQH_eqICcQETDcBC0X/view I think you can use this. Trained using FFHQ, 140k iter, 256px.

**Comment by anuragranj** [comment]
Thanks a lot @rosinality ! :)

**Comment by PgLoLo** [comment]
> @anuragranj https://drive.google.com/file/d/1SFn7GygaLYhOobNQH_eqICcQETDcBC0X/view I think you can use this. Trained using FFHQ, 140k iter, 256px.

I compared this weigts and weits that you gave in the issue #13 Are they the same? As far as I understand they are different both in quality and somehow in the architecture (maybe I'm wrong about architecture, but somewhy I get an error in to_rgb dimensions in my wrapper, but maybe it's my problem).

**Comment by PgLoLo** [comment]
In the #13 you stated that the checkpoint is made on the same 140k iterations. But comparing the same latent vector images are similar but different in quality:
from the #13 
![image](https://user-images.githubusercontent.com/5480856/57925423-e4388f80-78b0-11e9-8b54-d37c88626622.png)

from current issue:
![image](https://user-images.githubusercontent.com/5480856/57925449-f4506f00-78b0-11e9-9159-17c4b434dcb8.png)

**Comment by rosinality** [comment]
@PgLoLo https://drive.google.com/file/d/1SFn7GygaLYhOobNQH_eqICcQETDcBC0X/view is generator/discriminator checkpoints, and https://drive.google.com/file/d/1TVdUGOcMRVTVaxLhmh2qVmPgWIlwE0if/view is running average of generator weights. So they are different.

**Comment by PgLoLo** [comment]
@rosinality Oh, I see, thank you very much!

**Comment by ParthaEth** [comment]
Can you please upload some images while the training was happening. So like lower resolution images. e.g. 8X8 ones and then some 16X16 and so on. Mine look something like this
![image](https://user-images.githubusercontent.com/10944728/68139725-ebb92700-ff2a-11e9-97c5-53813a81ec90.png) at 8X8 then like this at 16X16 
![image](https://user-images.githubusercontent.com/10944728/68139822-1dca8900-ff2b-11e9-9d8b-5ab512ef2e83.png). I am trying to distill one with some conditioning over the shape and appearance and so on so the very similar looking ones are by design. :-) and no mode collapse. however higher up in the resolution I get weird looking ones like globally disfigured ones. Did this happen to any one else? e.g.
![image](https://user-images.githubusercontent.com/10944728/68140085-7f8af300-ff2b-11e9-882f-00a0842d44ca.png)

Any idea what might be going wrong?

**Comment by rosinality** [comment]
@ParthaEth Actually some weird samples can be happen as training steps between each phases are not that large. I think you can train more to get better results.

**Comment by ParthaEth** [comment]
You mean at every step before switching to next resolution?

**Comment by rosinality** [comment]
@ParthaEth Yes, you can train more at certain resolutions as training iterations during each phases are maybe not enough.
